<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecondWindow</class>
 <widget class="QDialog" name="SecondWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Выбор кораблей</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>361</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>22</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Бой против бота</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>181</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Корабли</string>
   </property>
   <widget class="QPushButton" name="btnShip1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1-палубный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShip2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>2-палубный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShip3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>3-палубный</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShip4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>4-палубный</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelShip1Count">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>1-палубные: 0/4</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShip2Count">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>2-палубные: 0/3</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShip3Count">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>3-палубные: 0/2</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelShip4Count">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>4-палубные: 0/1</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>160</y>
     <width>461</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayoutField"/>
  </widget>
  <widget class="QPushButton" name="btnStartBattle">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>В бой</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
